src/main/java/nu/studer/teamcity/buildscan/connection/DevelocityConnectionConstants.java

```diff
@@ -1,74 +1,74 @@
 package nu.studer.teamcity.buildscan.connection;
 
 @SuppressWarnings("unused")
-public final class GradleEnterpriseConnectionConstants {
+public final class DevelocityConnectionConstants {
 
-    // Constants defined by the Gradle Enterprise Connection
+    // Constants defined by the Develocity Connection
     // These are used to correlate data set by the user in the connection dialog to the descriptor parameters available in the Project Feature Descriptor Parameters
 
     public static final String GRADLE_PLUGIN_REPOSITORY_URL = "gradlePluginRepositoryUrl";
-    public static final String GRADLE_ENTERPRISE_URL = "gradleEnterpriseUrl";
+    public static final String DEVELOCITY_URL = "gradleEnterpriseUrl";
     public static final String ALLOW_UNTRUSTED_SERVER = "allowUntrustedServer";
-    public static final String GE_PLUGIN_VERSION = "gradleEnterprisePluginVersion";
+    public static final String DEVELOCITY_PLUGIN_VERSION = "gradleEnterprisePluginVersion";
     public static final String CCUD_PLUGIN_VERSION = "commonCustomUserDataPluginVersion";
-    public static final String GE_EXTENSION_VERSION = "gradleEnterpriseExtensionVersion";
+    public static final String DEVELOCITY_EXTENSION_VERSION = "develocityExtensionVersion";
```

Comment:
Was the value of this string changed intentionally? I notice others, such as `DEVELOCITY_PLUGIN_VERSION` were not changed.

These constants are eventually used by TC to persist the connection configuration to disk, so changing these could break the configuration of upgraders.

Reply:
You are right, I shouldn't have changed it. Reverted in https://github.com/etiennestuder/teamcity-build-scan-plugin/pull/373/commits/a58d23bfe53587904e44d9327fd563f7072041ed